src/neptune/metadata_containers/project.py

```diff
@@ -311,6 +318,9 @@ def fetch_models_table(
                     Namespaces: `["datasets", "info"]` - all the fields inside the namespaces are included as columns.
                 If `None` (default), all the columns of the models table are included.
             limit: How many entries to return at most (default: None - return all entries).
+            sort_by: Name of the column to sort the results by.
```

Comment:
Remember to call `verify_type("sort_by", sort_by, str)` for fetching models as well